app/serializers/concerns/subscribable.rb

```diff
@@ -0,0 +1,5 @@
+module Subscribable
+  def status
+    Subscription.for(discussion_id: object.id, user_id: scope.id).state
```

Comment:
There is mostly a naming issue here. This concern claims to make an object subscribable, but it only provides it's status, it does not allow to subscribe to it.

Moreover, it is coupled with it being a discussion, since the `for` method takes a `discussion_id`. Also, this method is called `status`, whereas in the subscription it's called `state`. That's not very consistent.